### PR TITLE
chore: remove non-null assertion operator

### LINKED ISSUE #368: (JS-0339) Found non-null assertions
## Description
Using non-null assertions cancels the benefits of the strict null-checking mode.

## Occurrences
There are 2 occurrences of this issue in the repository.

See all occurrences on DeepSource &rarr; [deepsource.io/gh/lenstube-xyz/lenstube/issue/JS-0339/occurrences/](https://deepsource.io/gh/lenstube-xyz/lenstube/issue/JS-0339/occurrences/)

### PR DESCRIPTION
**Pull request by zt4ff**
## Description

Closes #368 

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings

### CODE REVIEW

**Comment by sasicodes** [comment]
@zt4ff have added a comment

**Review by sasicodes — CHANGES_REQUESTED** [review]

File: packages/utils/functions/uploadToIPFS.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ export const everland = async (
       params
     })
     task.on('httpUploadProgress', (e) => {
-      const progress = ((e.loaded! / e.total!) * 100) | 0
+      const progress = ((e.loaded / e.total) * 100) | 0
```
[the remark this answers is not in the document]
Comment (zt4ff) [reply packages/utils/functions/uploadToIPFS.ts:46]: @sasicodes 
It is a TypeScript notation that tells the compiler that `.loaded` cannot be `null` or `undefined`

[the remark this answers is not in the document]
Comment (sasicodes) [reply packages/utils/functions/uploadToIPFS.ts:46]: can we use a separate variable for the loaded and total with nullish check?

loaded = e.loaded ?? 0
total = e.total ?? 0

**Review by sasicodes — CHANGES_REQUESTED** [review]

**Comment by sasicodes** [comment]
@zt4ff  have added comment 🙇🏻 
image

**Comment by sasicodes** [comment]
@zt4ff https://github.com/lenstube-xyz/lenstube/pull/398#issuecomment-1340520777

**Comment by sasicodes** [comment]
Fixed ✅

**Pull request closed without merging**